modular_zubbers/code/modules/antagonists/bloodsucker/powers/potence.dm

```diff
@@ -0,0 +1,67 @@
+
+/datum/action/cooldown/bloodsucker/potence
+	name = "Potence"
+	desc = "Endow yourself with extraordinary strength."
+	button_icon_state = "power_potence"
+	power_flags = BP_CONTINUOUS_EFFECT
+	check_flags = AB_CHECK_CONSCIOUS
+	purchase_flags = BLOODSUCKER_CAN_BUY|GHOUL_CAN_BUY
+	bloodcost = 10
+	constant_bloodcost = 1
+	cooldown_time = 20 SECONDS
+
+/datum/action/cooldown/bloodsucker/potence/get_power_explanation_extended()
+	. = list()
+	. += "Utilize your blood to augment your strength."
+	. += "This will drain blood at a constant rate of one unit per second while active."
+	. += "Your unarmed attacks will deal [GetPowerLevel()] more damage and gain [GetEffectiveness()] effectiveness."
+
+/datum/action/cooldown/bloodsucker/potence/ActivatePower(atom/target)
+	. = ..()
```

Comment:
as inconsistent as it is (i hate it) ActivatePower can be entirely overridene and has no functional code, but DeactivatePower does have functional code so keep calling and checking parent ther

Reply:
i don't know what this means i need it explained in byond babytalk

Reply:
Remove the . = ..(), it's not neccesary in ActivatePower